redaxo/src/core/lib/list.php

```diff
@@ -269,12 +276,9 @@ public function getCaption()
         return $this->caption;
     }
 
-    /**
-     * @param string $message
-     */
-    public function setNoRowsMessage($message)
+    public function setNoRowsMessage($msg)
```

Comment:
Scheinbar hast du Dateien aus einer älteren Version als Basis genommen und hier komplett reinkopiert, und so einige Änderungen im master wieder rückgängig gemacht.

Reply:
Was muss ich jetzt tun?

Reply:
Weiß jetzt nicht so genau, welches git-programm du nutzt, bzw. wie du arbeitest.
Ich würde:
- lokal nochmal frisch den master ziehen/aktualisieren
- daraus einen frischen neuen branch erstellen
- darin erneut deine änderungen machen
- den neuen lokalen branch zu deinem remote "rex_list"-branch force pushen
- damit überschreibst du den, und wir hätten hier im pr einen aktuellen stand

Aber wie gesagt, weiß nicht so genau, ob du git über ein Tool oder die cli verwendest, und weiß auch nicht, wie vertraut du mit dem thema so bist.

Alternativen wären:
- händisch hier alle änderungen rückgäng machen, die keine bewussten änderungen sind
- oder nochmal einen ganz neuen pr erstellen, basiert auf dem aktuellen master

Reply:
Oder arbeitest du direkt auf github?
Dann wäre vermutlich leichter einen neuen PR zu erstellen. Beginnen würde ich den hier im Repo im aktuellen master in der rex_list über den edit-button: https://github.com/redaxo/redaxo/blob/master/redaxo/src/core/lib/list.php